web/src/components/PcapOverIP.vue

```diff
@@ -0,0 +1,234 @@
+    >
+      <template #[`item.delete`]="props">
+        <v-tooltip bottom>
+          <template #activator="{ on, attrs }">
+            <v-btn
+              v-bind="attrs"
+              icon
+              v-on="on"
+              @click="
+                delDialogAddress = props.item.address;
+                delDialogVisible = true;
+              "
+            >
+              <v-icon>mdi-delete</v-icon></v-btn
+            >
+          </template>
+          <span>Delete</span>
+        </v-tooltip>
+      </template>
+    </v-data-table>
+    <v-dialog v-model="delDialogVisible" width="500" @keydown.enter="del">
+      <template #default
+        ><v-form>
+          <v-card>
+            <v-card-title class="text-h5">
+              Confirm PCAP-over-IP endpoint deletion
+            </v-card-title>
+            <v-card-text>
+              Do you want to delete the PCAP-over-IP endpoint
+              <code>{{ delDialogAddress }}</code
+              >?
+            </v-card-text>
+            <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn text @click="delDialogVisible = false">No</v-btn>
+              <v-btn
+                text
+                :disabled="delDialogLoading"
+                :loading="delDialogLoading"
+                :color="delDialogError ? 'error' : 'primary'"
+                @click="del"
+                >Yes</v-btn
+              >
+            </v-card-actions>
+          </v-card>
+        </v-form>
+      </template>
+    </v-dialog>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { EventBus } from "./EventBus";
+import ToolBar from "./ToolBar.vue";
+import { ref, computed, onMounted, onUnmounted } from "vue";
+import { useRootStore } from "@/stores";
+import moment from "moment";
+
+const delDialogVisible = ref(false);
+const delDialogAddress = ref("");
+const delDialogLoading = ref(false);
+const delDialogError = ref(false);
+
+const addDialogVisible = ref(false);
+const addDialogLoading = ref(false);
+const addDialogError = ref(false);
+const newAddress = ref("");
+const goodNewAddress = computed(() => {
+  return newAddress.value.match(/^.+:[0-9]+$/) != null;
+});
+
+const store = useRootStore();
+const headers = [
+  { text: "Address", value: "address", cellClass: "cursor-pointer" },
+  { text: "Status", value: "status", cellClass: "cursor-pointer" },
+  { text: "Packets Received", value: "packets", cellClass: "cursor-pointer" },
+  { text: "", value: "delete", sortable: false, cellClass: "cursor-pointer" },
+];
+
+const ticker = ref<NodeJS.Timeout | null>(null);
+const currentTime = ref(new Date().getTime());
+
+onMounted(() => {
+  ticker.value = setInterval(() => {
+    currentTime.value = new Date().getTime();
+  }, 1_000);
+});
+
+onUnmounted(() => {
+  if (ticker.value != null) clearInterval(ticker.value);
+});
+
+const items = computed(() => {
+  return store.pcapOverIPEndpoints?.map((e) => {
+    return {
+      address: e.Address,
+      status:
```

Comment:
Showing the plain connect/disconnect timestamp in the tooltip would be nice.

Reply:
done.